File: CMI/Contract/Order/Bestellhistorie.cs
using System;

namespace CMI.Contract.Order
{
    public class Bestellhistorie
    {
        public int AuftragsId { get; set; }
        public OrderType AuftragsTyp { get; set; }
        public ApproveStatus Freigabestatus { get; set; }
        public DateTime? DatumDerFreigabe { get; set; }
        public EntscheidGesuch EntscheidGesuch { get; set; }
        public DateTime? DatumDesEntscheids { get; set; }
        public string Besteller { get; set; }
        public string InterneBemerkung { get; set; }
        public string Sachbearbeiter { get; set; }
    }
}

File: CMI/Web.Clients/web-management/src/app/modules/ordermanagement/model/bestellhistorie.ts
import {
	ApproveStatus,
	EntscheidGesuchStatus,
	ShippingType
} from '@cmi/viaduc-web-core';

export class Bestellhistorie {
	public auftragsId: number;
	public auftragsTyp: ShippingType;
	public freigabestatus: ApproveStatus;
	public datumDerFreigabe: Date;
	public entscheidGesuch: EntscheidGesuchStatus;
	public datumDesEntscheids: Date;
	public besteller: string;
	public interneBemerkung: string;
	public sachbearbeiter: string;
}
